packages/styles/progressbar.css

```diff
@@ -1,13 +1,15 @@
 :root {
-  --progress-bar-background-color: var(--accent-dark);
-  --progress-bar-fill-color: var(--accent-info-light);
+  --progress-bar-background-color: var(--white);
+  --progress-bar-fill-color: var(--accent-primary);
+  --progress-bar-border-color: var(--gray-40);
   --progress-bar-animation-timing: 150ms;
 }
 
 .ProgressBar {
   background-color: var(--progress-bar-background-color);
-  padding: 2px;
+  padding: var(--space-half);
```

Comment:
This is an instance where we don't really want spacing to scale with the global padding variable. The padding in this instance is acting like a border, which we do want to have a fixed size. Additionally, I think the padding is wrong here. The thin/normal variants have a `16px` and `24px` footprint respectively, so this appears to be `3px` from the design.